build/gen_stub.php

```diff
@@ -533,7 +533,12 @@ public function toOptimizerTypeMask(): string {
     }
 
     public function toEscapedName(): string {
-        return str_replace('\\', '\\\\', $this->name);
+        // Escape \u and \U to avoid compilation errors in generated macros
+        return str_replace(
+            ['\\', '\u', '\U'],
```

Comment:
Maybe better consistently use double-backslashes?

```suggestion
            ['\\', '\\u', '\\U'],
```